sqlitecluster/SQLiteNode.cpp

```diff
@@ -1969,6 +1969,7 @@ void SQLiteNode::broadcast(const SData& message, SQLitePeer* peer) {
         SINFO("Sending broadcast: " << message.serialize() << " to peer: " << peer->name);
         _sendToPeer(peer, message);
     } else {
+        // TODO: do this in runOnAll
```

Comment:
I think the `TODO` is to remove this, right? I think the intention is to not use `SQLiteNode::broadcast` any longer.

Reply:
Oh you know what, I think we can actually remove the function now. We already refactored BedrockServer not to use this in broadcasting, and nothing else calls this directly, anything external like Auth uses `server->broadcastCommand()`